apps/andi/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Andi.MixProject do
   def project do
     [
       app: :andi,
-      version: "2.1.14",
+      version: "2.1.15",
```

Comment:
I assume CI/CD will flag this soon but, this needs to be .16 I think due to a recent merge